packages/shorebird_cli/lib/src/commands/account/create_account_command.dart

```diff
@@ -0,0 +1,98 @@
+import 'dart:async';
+
+import 'package:mason_logger/mason_logger.dart';
+import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/shorebird_config_mixin.dart';
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+
+/// {@template create_account_command}
+/// `shorebird account create`
+/// Create a new Shorebird account.
+/// {@endtemplate}
+class CreateAccountCommand extends ShorebirdCommand with ShorebirdConfigMixin {
+  /// {@macro create_account_command}
+  CreateAccountCommand({
+    required super.logger,
+    super.auth,
+    super.buildCodePushClient,
+  });
+
+  @override
+  String get description => 'Create a new Shorebird account.';
+
+  @override
+  String get name => 'create';
+
+  @override
+  Future<int> run() async {
+    final CodePushClient client;
+
+    if (!auth.isAuthenticated) {
+      try {
+        await auth.login(prompt, verifyEmail: false);
+        client = buildCodePushClient(
```

Comment:
why are we creating the client in this block? We're already creating it on line 47 so can we delete this?

Reply:
This has been deleted, resolving